codeforboston/maple
# Published testimony counter

# Summary
A fix for the view testimony pagination summary. Closes #1361.
Adds a call to firestore in the profile page to get the number published testimony of the user 
`/users/${profileprops.id}/publishedtestimony`
this is necessary because the usePublishedTestimonyListing hook which we use returns the testimony page by page, without a total. 

# Screenshots

<img width="411" alt="image" src="https://github.com/codeforboston/maple/assets/30247522/1a2565a9-410b-455f-84d3-6b5940ce2d0b">
# Known issues
It's possible that this action should be refactored as part of usePublishedTestimonyListing or another hook. 


# Steps to test/reproduce

1. Go to a profile page
1. See the summary of testimony pages -- eg `viewing 21-30 of 45` 
1. Page through the testimony and see if the numbers are correct
